assets/runtime/functions

```diff
@@ -237,7 +237,16 @@ gitlab_uninstall_unused_database_client() {
     REGEX_DB_CLIENT_VERSIONS_IN_USE="-common${DB_CLIENT_VERSIONS_IN_USE}"
 
     # remove unused client using regex above
-    UNUSED_DB_CLIENTS=$(apt-cache pkgnames postgresql-client | grep -v -e "${REGEX_DB_CLIENT_VERSIONS_IN_USE}" | tr '\n' ' ')
+    # grep may return non-zero code on mo match, so fake the exit code with the `|| true` to swallow that
+    UNUSED_DB_CLIENTS=$(apt-cache pkgnames postgresql-client | grep -v -e "${REGEX_DB_CLIENT_VERSIONS_IN_USE}" || true)
```

Comment:
You don't care about newlines anymore. Does the `apt-get purge` command works in as expected?

Reply:
Yes of course I have made sure.

And, `tr` command to convrert newline in `UNUSED_DB_CLIENTS` is moved before `purge`. Could you check [line 248](https://github.com/sameersbn/docker-gitlab/pull/2774/files#diff-934fe23ac2af75a265cbd6e8da77882402932d36cfe726f9b11f1a43d0bdc5bfR248)?

Reply:
Ooops, I missed that line. 

---

I'm glad you're safe. I hope no major damage was caused by the typhoon.